content/docs/command-reference/move.md

```diff
@@ -71,6 +71,17 @@ outs:
     md5: c8263e8422925b0872ee1fb7c953742a
     path: other.csv
 ```
+Note that when we try to use `dvc move` over a file whose pattern matches one of the patterns listed in `.dvcignore`, it would raise an error because that DVC-file was not tracked by DVC.
+
+```dvc
+$ dvc add data.csv
+$ echo data.* >> .dvcignore
+$ dvc move data.csv other.csv
+ERROR: failed to move 'data.csv' -> 'other.csv' - Unable to find DVC-file with output 'data.csv'
+
+Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
+```
+See [.dvcignore](docs/user-guide/.dvcignore) for more details.
```

Comment:
This also looks good to me, maybe just put it under an H2 heading such as `## Dvcignore effects`. Same in other docs.

Reply:
Actually it's probably best without header and also without example. Just a very short note should do the trick here. We just want users to be aware of .dvcignore, not to explain it in every command ref.